packages/portal/src/cachers/matomo/visits.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+const LOCALISE = false;
+const PICK = false;
+
+const data = async(config = {}) => {
+  const response = await axios.get('/', {
+    baseURL: config.matomo.host,
+    params: {
+      date: 'last30',
+      format: 'JSON',
+      idSite: config.matomo.siteId,
+      method: 'VisitsSummary.get',
+      module: 'API',
+      period: 'day',
+      'token_auth': config.matomo.authToken
+    }
+  });
+
+  const totalVisits = Object.keys(response.data)
```

Comment:
When there is no response data because for example the connection failed or the auth token doesn't work this will set the totalVisists to 0.
This may look odd on the promo page, shouldn't we check for a successful response and only update the number when some data came back?

Reply:
Yep. Handling has been added.